decidim-elections/db/migrate/20210330123606_add_voting_ballot_style.rb

```diff
@@ -4,7 +4,7 @@ class AddVotingBallotStyle < ActiveRecord::Migration[5.2]
   def change
     create_table :decidim_votings_ballot_styles do |t|
       t.string :title
-      t.string :code
+      t.string :code, index: true
```

Comment:
💬 Intead of having two independent indexes for the voting and the code you can define only one with two columns (voting first and code then), make it unique for the previous comment, and then be sure that all the queries search always by voting (and then by code if needed) to ensure that they are used.

Reply:
https://github.com/decidim/decidim/pull/7779/commits/92f64a294bb5507090486a9b351298b18ea43069